src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,24 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  padding: 50px 0 73px;
+
+  &__container {
+    width: 1024px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
```

Comment:
Currently your container's always 1024px. Have you checked how it will work if the number of cards increases?

Checklist 13